src/js/components/forms/Dropdown.jsx

```diff
@@ -56,6 +56,18 @@ export class Dropdown extends React.Component {
         return this.state.selectedOption;
     }
 
+    selectOption(option) {
```

Comment:
We could extract the current behaviour into a fairly trivial wrapper component which just provides the state, exported but marked for future deletion, and update existing client code to use that instead, and mark it with TODOs to switch to the new one?

Too many commas :D